src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/ProcessorTestSuite.java

```diff
@@ -80,6 +81,9 @@
     @Autowired
     protected ProcessEventRepository processEventRepository;
 
+    @Autowired
+    OcrValidationRetryManager ocrValidationRetryManager;
```

Comment:
access modifier

Reply:
`protected` are actually not necessary, maybe worth removing all of them in one separate PR.